site/server/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,16 @@
+# --- !Ups
```

Comment:
Definitely, I'm going to squash all the evolutions into one with the final schema.